src/network/networkpacket.cpp

```diff
@@ -81,6 +81,24 @@ void NetworkPacket::putRawString(const char* src, u32 len)
 	m_read_offset += len;
 }
 
+void NetworkPacket::putCString(const char* src, u32 len)
+{
+	u16 msgsize = len;
+	if (msgsize > 0xFFFF) {
+		msgsize = 0xFFFF;
```

Comment:
Won't the compiler just optimize the *if* away because an u16 can't be `> 0xffff`?